index.bs

```diff
@@ -798,17 +816,25 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 		{{Clipboard/read()}} returns a {{Promise}} to [=clipboard items=] object that represents contents of [=system clipboard data=].
 		</p>
 
+		{{ClipboardUnsanitizedFormats/unsanitized}} is a [=sequence=] of {{DOMString}}s corresponding to the [=representation/mime type=] that are in the [=unsanitized data types=].
```

Comment:
```suggestion
		{{ClipboardUnsanitizedFormats/unsanitized}} is a [=sequence=] of {{DOMString}}s corresponding to the [=representation/mime type=] that should be handled as [=unsanitized data types=].
```

Reply:
`Unsanitized data types` is a list of MIME types that must not be sanitized. It doesn't describe any processing steps. If the web authors specify these MIME types in the `{{ClipboardUnsanitizedFormats/unsanitized}}`, then we will handle that according to the steps in `read` method.